sdk/provisioning/Azure.Provisioning.CloudMachine/tests/TdkTests.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System.Threading.Tasks;
+using Azure.Provisioning.CloudMachine;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using NUnit.Framework;
+
+namespace Azure.CloudMachine.Tests;
+
+public class TdkTests
+{
+    [Theory]
+    [TestCase([new string[] { "-tsp" }])]
+    public void BasicTsp(string[] args)
```

Comment:
Can we add an Assert on the result of the generation somehow?

Reply:
will do